src/library/scala/collection/immutable/StringLike.scala

```diff
@@ -66,8 +66,9 @@ self =>
   /** Return the current string concatenated `n` times.
    */
   def * (n: Int): String = {
+    val str = toString
```

Comment:
You should special case the 0 case - then you can avoid the buffer creation to toString entirely,

Reply:
0 and 1

Reply:
I will address that in #33